src/main/resources/application.conf

```diff
@@ -186,6 +189,13 @@ collector {
       timeLimit = 1000
     }
   }
+
+  enableDefaultRedirect = false
+  redirectDomains     = []
+  enableStartupChecks = true
+  terminationDeadline = 10.seconds
+  preTerminationPeriod = 10.seconds
```

Comment:
I suggest 0 seconds.  There's no need for micro to delay shutting down.  A 10 second delay will quickly get annoying for users.